.github/scripts/svgo.config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
```

Comment:
Shouldn't we provide this on the root directory? I think it'd be better there, IMO.

Reply:
In my point of view I don't see why, since this script is related to a specific workflow, differently, for instance, of `gulpfile.js` that is related to the whole project. 🤔

Reply:
Are we talking about the file location? If so I agree with @lunatic-fox. Scripts only used in the workflow should be stored in the workflow/script folder :)